t/cli/test_standalone.sh

```diff
@@ -65,3 +65,139 @@ if [ ! $code -eq 200 ]; then
 fi
 
 echo "passed: resolve variables in apisix.yaml conf success"
+
```

Comment:
This file is not suitable to add this test.
Please add it in node/upstream.t like https://github.com/apache/apisix/blob/a490e0a2570d916921b4a4e50f3c308291ba7dd9/t/node/upstream.t#L400

Reply:
Fixed. Please review it again.